apollo-client/src/main/java/com/ctrip/framework/apollo/internals/SimpleConfig.java

```diff
@@ -111,7 +123,7 @@ public String apply(ConfigChange input) {
     updateConfig(newConfigProperties, m_configRepository.getSourceType());
     clearConfigCache();
 
-    this.fireConfigChange(m_namespace, changeMap);
+    this.fireConfigChange(m_appId, m_namespace, changeMap);
```

Comment:
Here use the m_appId of current object, rather than the method passed appId, is I imitate your m_namespace writing, determine to change to the passed appId?

Reply:
Although the `appId` should be the same as the `m_appId`, I think it's better to use `appId` as it is passed in as a parameter.